frontend/packages/helm-plugin/src/components/details-page/resources/HelmReleaseResources.tsx

```diff
@@ -29,8 +29,8 @@ const HelmReleaseResources: React.FC<HelmReleaseResourcesProps> = ({ customData
       return {
         ...(model.namespaced ? { namespace } : {}),
         kind: model.crd ? referenceForModel(model) : model.kind,
-        name: resource.metadata.name,
-        prop: `${resource.metadata.name}-${resource.kind.toLowerCase()}`,
+        name: resource.metadata?.name,
+        prop: `${resource.metadata?.name}-${resource.kind?.toLowerCase()}`,
```

Comment:
If these values aren't set, won't this cause a problem? We should handle those cases if they might actually be undefined.